Source/src/main/java/CyberPunkReas/CovidSimulator/models/Restriction.java

```diff
@@ -1,31 +1,41 @@
-package CyberPunkReas.CovidSimulator.models;
+package Master.Design_Models;
 
 public class Restriction {
 
     private boolean curfew;
-    private boolean businessEssential;
-    private boolean businessNonEssential;
+    private boolean businessEssentialClosed;
+    private boolean businessNonEssentialClosed;
     private int timeRestriction;
-
+    private boolean mobility ;
+    
     public Restriction() {
         curfew = false;
-        businessEssential = false;
-        businessNonEssential = false;
+        businessEssentialClosed = false;
+        businessNonEssentialClosed = false;
         timeRestriction = 0;
+        mobility =false;
+    }
+    // Constructor for mockup
+    public Restriction(boolean curfew, boolean businessE,boolean businessNE,int timeRestriction, boolean mobility) {
```

Comment:
try to not "compress" names. E and NE could mean anything and we couldnt tell without context (by opening the class or knowing what's inside) what that means